apps/private-law/prl-cos/ithc-image-policy.yaml

```diff
@@ -3,5 +3,11 @@ kind: ImagePolicy
 metadata:
   name: ithc-prl-cos-api
 spec:
+  filterTags:
```

Comment:
you need to add, disable prod automation

Reply:
@AlokDatta added annotation to disable prod deployment check now and approve